hedyorg/hedy
# [FIX] Fix customization showing wrong first level

The customizations page recently was modified to show by default the minimum available level. It caused an error showing the wrong minimum level when in the database the levels were not stored in order. Now, instead of comparing to the loop index, we compare to the level number.

**Fixes #4178**

**How to test**

Go to dev_database.json and change the order of the field sorted_adventures levels and put some other level first. It should still show the minimum level correctly.
